charter.md

```diff
@@ -53,15 +53,227 @@ The following responsibilities belong to the Steering Committe, but may be deleg
 * Control and delegate access to and establish processes regarding
   project resources/assets<sup>[6](#footnote6)</sup>  
 
-## Quorum
+## Changes
 
-Steering committee members are generally expected to attend every meeting. We
+In instances where a process is not already specified within this document,
+changes to the Steering Committee charter will be considered according to the
+processes set forth in the committee's [operations documentation][changes].
+
+[changes]: /operations/changes.md
+
+## Membership
+
+### Composition
+
+The Steering Committee is composed of seven (7) members.
+
+### Elections
+
+<!--
+
+- Consider explicitly mentioning SIG ContribEx's Elections subproject, once it
+exists.
+
+-->
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+In the instance that a general election is called as a result of the
+[dissolution](#dissolution) of the current committee, the following rules will
+apply:
+
+* members of the current committee will not be allowed to run in the new
+  election
+* a new committee will be bootstrapped with:
+  * the top four candidates serving a two-year term
+  * the next three candidates serving a one-year team
+
+Policy and operations of special elections initiated by the Steering Committee
+for other reasons (as yet to be determined) may be handled by some agreed-upon
+[process][changes] by the committee.
+
+[general-elections]: /elections.md
+[sig-contribex]: https://git.k8s.io/community/sig-contributor-experience
+
+### Vacancies
+
+In the event of a resignation or other loss of an elected steering committee
+member, the candidate with the next most votes from the previous election will
+be offered the seat.  This process will continue until the seat is filled.
```

Comment:
It makes sense to fill the seat if the candidates below received a good number of votes (ie top 2-3 runners up with X% of the vote min) but doesn't quite makes sense if you just go down a list infinitely and appoint someone with barely any votes instead of just having a new election.

Reply:
(Realizing that this is the existing policy but my comment stands esp given the new additions to the document)

Reply:
3 or 4 people have said similarly, but the old comment seems to have gotten detached from the current revision.

Reply:
Linking @jberkus' [review](https://github.com/kubernetes/steering/pull/248#discussion_r914285986) to reconnect the history:

```suggestion
Committee member, the next most preferred candidate from the previous 
```

> Per below, I'd like to hold off on revising this section because I wanted to do it through the Elections Subproject.  However, if we are revising it, the phrase "next most votes" needs to go away, because it's not accurate.  We use preference elections.


@parispittman:

> agree with you. lets not change this language right now.

Reply:
This section didn't matter before since there was basically no way to get a vacancy, but now there are several ways. I think you can fix this with a single additional line, "A maximum of one board member may be selected this way between elections." I would add more, but I think that's the minimum right now.

Reply:
Oh, I see what happened -- @justaugustus that's not the right comment thread, let me find it for you.

Reply:
Thanks!

Reply:
Marking this as resolved.